configure

```diff
@@ -436,6 +448,11 @@ if (exists($opts{'std'})) {
   # Accept any of --std=17, --std=stdcpp17, --std=c++17, etc.
   $cxx_std =~ s/^(std)?(cpp|c\+\+)//;
 }
+my @mpcopts;
+push(@mpcopts, @{$opts{'mpcopts'}}) if exists($opts{'mpcopts'});
+my @features;
+push(@features, @{$opts{'features'}}) if exists($opts{'features'});
+push(@features, 'java=1') if $opts{'java'};
```

Comment:
Can this be set where other Java-related settings are handled?  If it has to be here, maybe add a comment.

Reply:
I put it here to be safe, but looking over how `features` is used, I think it can go with the `java_pre_jpms` code.